### PR TITLE
[NCL-6165] pass restMode to the adjust tools

### PR DESCRIPTION
**Pull request by janinko**
### All Submissions:

* [ ] Have you added a note in the [CHANGELOG wiki](https://github.com/project-ncl/repour/wiki/Changelog) for your change?
* [ ] Have you added unit tests for your change?

### CODE REVIEW

**Comment by thescouser89** [comment]
@janinko could you please run:
```
poetry run tox
```
twice. The first time you run it, it should fail on the 'black' part for the formatting. 'black' is used as an opinionated code formatter for our codebase. The second time you run it, it'll pass and go to the tests (that probably will fail because you have Python 3.8 and right now Repour only supports Python 3.6)

That should make the CI tests happier afterwards. Thanks!

File: repour/adjust/project_manipulator_provider.py [hunk 1]
```diff
@@ -100,12 +101,12 @@ async def adjust(work_dir, extra_adjust_parameters, adjust_result):
...
             )
             temp_suffix = ("-" + orig_inc_suffix) if orig_inc_suffix else ""
-            temp_build_parameters.append(
+            alignment_parameters.append(
                 "-DversionIncrementalSuffix=" + suffix_prefix + temp_suffix
             )
 
-        if specific_indy_group:
-            temp_build_parameters.append("-DrepositoryGroup=" + specific_indy_group)
+        if brew_pull_enabled:
```
Comment (pkocandr) [inline repour/adjust/project_manipulator_provider.py:108]: This should not be here for Project Manipulator. NPM packages cannot be pulled from Brew, they are from PNC only

File: repour/adjust/util.py [hunk 2]
```diff
@@ -71,61 +72,37 @@ def is_temp_build(adjustspec):
...
-    temp_build_enabled = is_temp_build(adjustspec)
-
-    if temp_build_timestamp_key in adjustspec:
-        suffix_prefix = adjustspec[temp_build_timestamp_key]
+    suffix_prefix = build_category_config.suffix_prefix
 
     if temp_build_enabled:
-        if suffix_prefix is None:
-            logger.info("Temp build timestamp set to: " + str(suffix_prefix))
+        if not suffix_prefix:
```
Comment (pkocandr) [inline repour/adjust/util.py:102]: Invert the if to drop the "not" ;-)

**Review by pkocandr — COMMENTED** [review]
Apart from sending non-existing brewPullActive param into Project Manipulator it looks good.

**Review by thescouser89 — APPROVED** [review]

**Pull request merged**